site/guides/producer/setters/README.md

```diff
@@ -53,7 +53,7 @@ in this guide.
 3. Locate all fields which reference the setter and change their values.
 4. Write both the modified Kptfile and resources back to the package.
 
-{{< svg src="images/set-command" >}}
+![img](/../../../static/images/set-command.svg)
```

Comment:
can this be normalized into an absolute link? Can be a follow up.

Reply:
Done.